.github/workflows/graphqlmetrics-ci.yaml

```diff
@@ -12,6 +12,7 @@ concurrency:
 env:
   CI: true
   INT_TESTS: true
+  GOWORK: 'off'
```

Comment:
You would also need to update "make lint" in each go project.

Reply:
Thanks for the hint.